packages/jumpstarter/jumpstarter/common/utils.py

```
@@ -76,6 +75,11 @@ def launch_shell(
         if cmd[0].endswith("bash"):
             cmd.append("--norc")
             cmd.append("--noprofile")
+            env["PS1"] = f"{ANSI_GRAY}{PROMPT_CWD} {ANSI_YELLOW}⚡{ANSI_WHITE}{context} {ANSI_YELLOW}➤{ANSI_RESET} "
+        elif cmd[0].endswith("zsh"):
+            cmd.append("-f")
+            cmd.append("-i")
+            env["PROMPT"] = "%F{grey}%~ %F{yellow}⚡%F{white}local %F{yellow}➤%f "
 
         process = Popen(cmd, stdin=sys.stdin, stdout=sys.stdout, stderr=sys.stderr, env=env)
```

Pylint (3.3.7)

[refactor] Consider using 'with' for resource-allocating operations

(R1732)